specs/darwin/browser_plugins.table

```diff
@@ -1,5 +1,5 @@
 table_name("browser_plugins")
-description("All C/NPAPI browser plugin details for all users.")
+description("All C/NPAPI browser plugin details for the user context. C/NPAPI has been deprecated on all major browsers. To query for plugins on modern browsers, try: `chrome_extensions` `firefox_addons` `safari_extensions`.")
```

Comment:
I'm not sure what "for the user context means", so I don't think this adds clarity. 

As a side note, I think the safari_extension table might also be somewhat out of date

Reply:
Yeah, it's not something I know exactly how to communicate. 

What I'm trying to convey with the "user context" thing is that user_data=True in the attributes below. I think userData=True means that it's untrue that it returns "plugin details for all users" as it currently says, right? 

Not exactly sure how to communicate that clearly... The osquery wiki states: 
```
**user_data=True**: This tells the caller that they should provide a `uid` in the query predicate. By default the table will inspect the current user's content, but may be asked to include results from others.
```

Maybe to be standard, we ought to use that language here? "plugin details for the current user's content."

Reply:
Any idea what's wrong with the safari table? Could be documented too.

Reply:
I don't think we should mention anything about user_data here. It feels too ad-hoc. I think it would make more sense to figure out a uniform approach

Reply:
OK I will change